import-va-certs.sh

```diff
@@ -4,7 +4,10 @@ set -euo pipefail
 
 (
     cd /usr/local/share/ca-certificates/
-    
+
+    curl -LO https://cacerts.digicert.com/DigiCertTLSRSASHA2562020CA1-1.crt.pem
```

Comment:
Are these additional certs that we need on top of the ones we import from http://aia.pki.va.gov/PKI/AIA/VA/?

Reply:
This is one of the certs from the SecretStore:` DigiCertTLSRSASHA2562020CA1.crt.pem` [here](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/main/apps/vets-api/prod/templates/secrets.yaml#L984)

Reply:
Is this file necessary?

Reply:
import-va-certs.sh matches the file in k8s.

Reply:
@LindseySaari @stevenjcumming

Reply:
Kyle added these changes a while ago. It should be included in master (as this PR is doing).